packages/ag-charts-community/src/scene/canvas/hdpiCanvas.ts

```diff
@@ -99,7 +99,7 @@ export class HdpiCanvas {
 
     private onEnabledChange() {
         if (this.element) {
-            this.element.style.display = this.enabled ? 'block' : 'none';
+            this.element.style.display = this.enabled ? '' : 'none';
```

Comment:
We need the canvas to be `display: block` because it defaults to `inline-block`. Not setting it can mess up the alignment and outer height

Reply:
This is being set in the CSS managed by the DOMManager by default.